.github/workflows/dependency-tree-update.yml

```diff
@@ -45,6 +45,8 @@ jobs:
         with:
           registry-url: 'https://npm.pkg.github.com'
       - name: Update dependency tree.
+        id: update
+        continue-on-error: true
```

Comment:
Does the step still look "failed" in the actions UI? I think that's a nice detail if we can keep that, so it's easier to find the error.
Otherwise, I think we can use different conditions in the `if`s. There's an `if: failure()` that we can put on the issue step instead. I guess actually `if: ${{ failure() && !cancelled() }}` :thinking:

Reply:
✅